docs/lesiaturuta/5-logic/homework.js

```diff
@@ -0,0 +1,119 @@
+/**
```

Comment:
Hey, @lesiaturuta, would be better to fully integrate this part into your project. 
```
lesiaturuta (root folder)
- assets (images folder)
- - card.png
- - all other images
- styles (all css files inside)
- scripts
- - pokemon.js
- - calculator.js
- - all other scripts
- index.html (renamed about.html)
- login.html
- calculator.html
- ...
```

Reply:
fixed

Reply:
Please rename folder `loader` to `styles` and move there main css file